src/main/javascript/src/views/ExperimentSummary.vue

```diff
@@ -51,41 +320,291 @@
+      },
+      {
+        title: "Participants",
+        description:
+          "How students in your class become participants in your experiment",
+        image: require("@/assets/participants_summary.svg"),
+      },
+      {
+        title: "Assignments",
+        description: "All experiment assignments",
+        image: require("@/assets/assignments_summary.svg"),
+      },
+    ],
+    conditionTreatments: {},
+  }),
+  methods: {
+    ...mapActions({
+      fetchExposures: "exposures/fetchExposures",
+      checkTreatment: "treatment/checkTreatment",
+      createTreatment: "treatment/createTreatment",
+      createAssessment: "assessment/createAssessment",
     }),
-    methods: {
+    // Navigate to EDIT section
+    handleEdit(componentName) {
+      this.$router.push({ name: componentName });
+    },
+    async getAssignmentDetails() {
+      await this.fetchExposures(this.experiment.experimentId);
+      return this.exposures;
+    },
+    hasTreatment(conditionId, assignmentId) {
+      const assignmentBasedOnConditions = this.conditionTreatments[
+        +conditionId
+      ];
 
+      return (
+        assignmentBasedOnConditions?.find(
+          (assignment) => assignment.assignmentId === assignmentId
+        ) !== undefined
+      );
     },
-    created() {
-      this.tab = (this.$router.currentRoute.name==='ExperimentSummary')? 1 : 0
+    async handleCreateTreatment(conditionId, assignmentId) {
+      // POST TREATMENT
+      try {
+        return await this.createTreatment([
+          this.experiment.experimentId,
+          conditionId,
+          assignmentId,
+        ]);
+      } catch (error) {
+        console.error("handleCreateTreatment | catch", { error });
+      }
     },
-    beforeRouteEnter (to, from, next) {
-      return store.dispatch('experiment/fetchExperimentById', to.params.experiment_id).then(next, next)
+    async handleCreateAssessment(conditionId, treatment) {
+      // POST ASSESSMENT TITLE & HTML (description)
+      try {
+        return await this.createAssessment([
+          this.experiment.experimentId,
+          conditionId,
+          treatment.treatmentId,
+        ]);
+      } catch (error) {
+        console.error("handleCreateAssessment | catch", { error });
+      }
     },
-    beforeRouteUpdate (to, from, next) {
-      return store.dispatch('experiment/fetchExperimentById', to.params.experiment_id).then(next, next)
+    async goToBuilder(conditionId, assignmentId) {
+      // create the treatment
+      const treatment = await this.handleCreateTreatment(
+        conditionId,
+        assignmentId
+      );
+      // create the assessment
+      const assessment = await this.handleCreateAssessment(
+        conditionId,
+        treatment?.data
+      );
+
+      // show an alert if there's a problem creating the treatment or assessment
+      if (!treatment || !assessment) {
+        alert("There was a problem creating your assessment");
```

Comment:
All alerts and confirmation dialogues need to use SweetAlert2. You can do this here by converting to `this.$swal('There was a problem creating your assessment')`

Reply:
Updating